### ISSUE TITLE
udev: disks used as LVM PV no longer show the disk model in ID_MODEL but LVM PV information instead

### ISSUE THREAD

**Issue by emmericp** [lvmteam/lvm2#21]
```
$ udevadm info /dev/sdX
E: ID_MODEL=LVM PV vBAYkl-gxgV-AZ2n-pgix-pI6y-6cTD-PUxl4C on /dev/sdc
E: ID_MODEL_ENC=WDC\x20WD3000BLFS-01YBU0\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20
```

It was much more helpful to show the actual disk model in ID_MODEL. Also, I'd expect ID_MODEL == trim(decode(ID_MODEL_ENC))

Introduced here: https://github.com/lvmteam/lvm2/commit/99bfbbf229acf4548f1ffc06625f464dc0ae4ca4

Which I guess is somewhat intentional because it explicitly sets ID_MODEL, but the commit mentions "This patch introduces no functional change to the udev rules." 

See also: https://github.com/systemd/systemd/issues/12947

**Comment by prajnoha** [comment]
The ENV{ID_MODEL} assignment for disks which are PVs is quite old, it goes back to the early versions of lvm-systemd-udev interaction and introduced with the patch 3fee6610286f83737d193c9047e24e8d7c85d1ce (~ 6 years ago).

This was done for the messages in the logs to be more informative and straightforward to search PVs:

```
#systemctl -a | grep "LVM PV"
  dev-block-252:2.device                                                                                         loaded    active   plugged   LVM PV JhxC7B-YTgk-3jIU-5GVo-c4gV-W8t3-UUz06p on /dev/vda2 2        
  dev-disk-by\x2did-lvm\x2dpv\x2duuid\x2dJhxC7B\x2dYTgk\x2d3jIU\x2d5GVo\x2dc4gV\x2dW8t3\x2dUUz06p.device         loaded    active   plugged   LVM PV JhxC7B-YTgk-3jIU-5GVo-c4gV-W8t3-UUz06p on /dev/vda2 2      
```

But I think we can live without that and we can remove the ID_MODEL part so we don't overwrite the actual ID_MODEL if it exists. There's also the /dev/disk/by-id entry for each PV anyway as seen from the example above...

Patched with fef8e506891bd159475de83f248fdac660675163 (master branch) and 1650c104384b81acd5b2e30f507624f98309f2ff (stable-2.02 branch).

**Comment by emmericp** [comment]
thanks for the quick fix :)
